src/OpenRiaServices.Tools/Test/ClientCodeGenerationDispatcherTests.cs

```diff
@@ -555,6 +555,15 @@ public void ClientCodeGenerationDispatcher_Error_TypeLoadException()
 
                 Assert.IsTrue(File.Exists(typeLoadExceptionProjectLocation), "Expected TypeLoadExceptionProject.dll to coreside with this assembly in test folder");
 
+                using (var assemblyDefinition = AssemblyDefinition.ReadAssembly(typeLoadExceptionProjectLocation, new ReaderParameters { ReadWrite = true }))
+                {
+                    var module = assemblyDefinition.MainModule;
+                    var class1 = module.Types.Single(t => t.Name == "Class1");
+                    var class2 = module.Types.Single(t => t.Name == "Class2");
+                    class2.BaseType = class1.BaseType;
+                    assemblyDefinition.Write();
```

Comment:
Can the reuse of the same .dll give problems if the test is run multiple times? 
- should the modified assembly have another name ?